Rb.Forms.Barcode.Droid/BarcodeScannerRenderer.cs

```diff
@@ -2,104 +2,120 @@
 using System.ComponentModel;
 using Xamarin.Forms;
 using Xamarin.Forms.Platform.Android;
-
 using Rb.Forms.Barcode.Pcl;
 using Rb.Forms.Barcode.Pcl.Logger;
 using Rb.Forms.Barcode.Droid;
 using Rb.Forms.Barcode.Droid.Camera;
-using Rb.Forms.Barcode.Droid.Decoder;
 using Rb.Forms.Barcode.Droid.Logger;
-
 using Android.Views;
 using System.Threading.Tasks;
 using Rb.Forms.Barcode.Droid.View;
+using Android.Content.PM;
+using Android.Runtime;
```

Comment:
For future stuff, you could use Xamarin Studio's _order usings_ quick refactor so that using are grouped by namespace.  This can be turned enabled by turning on Source Analysis:

<img width="769" alt="3" src="https://cloud.githubusercontent.com/assets/624632/10448348/920745b4-718c-11e5-99b9-d159b8b7c349.png">

And then opening the context-menu over the usings:

<img width="617" alt="4" src="https://cloud.githubusercontent.com/assets/624632/10448364/ac1c0304-718c-11e5-9843-1248cca4a324.png">